src/main/kotlin/no/nav/familie/ba/sak/behandling/domene/BehandlingRepository.kt

```diff
@@ -16,7 +16,7 @@ interface BehandlingRepository : JpaRepository<Behandling, Long> {
     @Query("SELECT b FROM Behandling b JOIN b.fagsak f WHERE f.id = :fagsakId AND b.gjeldendeForUtbetaling = true")
     fun findByFagsakAndGjeldendeForUtbetaling(fagsakId: Long): List<Behandling>
 
-    @Query("SELECT b FROM Behandling b JOIN b.fagsak f WHERE f.id = :fagsakId AND b.status in ('IVERKSATT', 'FERDIGSTILT')")
-    fun findByFagsakAndFerdigstiltOrIverksatt(fagsakId: Long): List<Behandling>
+    @Query("SELECT b FROM Behandling b JOIN b.fagsak f WHERE f.id = :fagsakId AND b.status in ('AVSLUTTET')")
```

Comment:
```suggestion
    @Query("SELECT b FROM Behandling b JOIN b.fagsak f WHERE f.id = :fagsakId AND b.status = 'AVSLUTTET'")
```
Trur man kan skrive no sånt her?